### PR TITLE
TCS-164 - WDB Datastriping Check

### PR DESCRIPTION
**Pull request by ShawnTay96**
[[TCS-164 - WDB Datastriping Check](https://torqv5cloudscaling.atlassian.net/jira/software/projects/TCS/boards/1/backlog?selectedIssue=TCS-164)]
WDB should not spin up when datastriping is on

### CODE REVIEW

File: code/processes/wdb.q [hunk 1]
```diff
@@ -18,7 +18,7 @@ upd:.wdb.replayupd;
 /- initialise the wdb process
 .wdb.startup[];
 / - start the timer if datastriping off
```
Comment (dmitchard) [inline code/processes/wdb.q:20]:
```suggestion
/ - start the timer if datastriping off or exit process if datastriping on
```

**Review by dmitchard — APPROVED** [review]

**Review by THarveyWork — COMMENTED** [review]
Looks good!

**Review by ccrease01 — APPROVED** [review]
Looks good

File: code/processes/wdb.q [hunk 2]
```diff
@@ -17,8 +17,8 @@ upd:.wdb.replayupd;
 .wdb.clearwdbdata[];
 /- initialise the wdb process
 .wdb.startup[];
-/ - start the timer if datastriping off
-$[.ds.datastripe;.lg.o[`data;"datastriping on - savedown to ",(string .wdb.savedir)," disabled"];.wdb.saveenabled;.wdb.starttimer[]];
+/ - start the timer if datastriping off or exit process if datastriping on
+$[.ds.datastripe;[.lg.o[`data;"datastriping on - wdb will not start up "];exit 0];.wdb.saveenabled;.wdb.starttimer[]];
```
Comment (jonathonmcmurray) [inline code/processes/wdb.q:21]: can you just make this a `.lg.e`? that should exit proc if encountered while proc is initialising

```suggestion
$[.ds.datastripe;.lg.e[`data;"datastriping on - wdb will not start up "];.wdb.saveenabled;.wdb.starttimer[]];
```

File: tests/wdbcheck/datastripeoff/tests.csv [hunk 3]
```diff
@@ -0,0 +1,2 @@
+action,ms,bytes,lang,code,repeate,minver,comment
+true,0,0,q,any@[hopen;`::36005:admin:admin;0],1,,"wdb should start up when datastriping is off"
```
Comment (jonathonmcmurray) [inline tests/wdbcheck/datastripeoff/tests.csv:2]: I don't think there should be hardcoded ports & user/pass in the unit tests, that makes these pretty config-specific.

Might be best to use a system command to check if revelant proc is running

(same applies to other test case)

**Review by jonathonmcmurray — APPROVED** [review]

**Pull request merged**